README.md

```diff
@@ -546,6 +547,26 @@ Additionally an optional string parameter `system_prompt` can be specified in `/
 
 Activate token quota limits for the service by adding a new configuration structure into the configuration file. That structure should be added into `ols_config` section. It has the following format:
 
+
+
+### Tokens and token quota limits
+
+Tokens are small chunks of text, which can be as small as one character or as large as one word. Tokens are the units of measurement used to quantify the amount of text that the service sends to, or receives from, a large language model (LLM). Every interaction with the Service and the LLM is counted in tokens.
+
+Token quota limits define the number of tokens that can be used in a certain timeframe. Implementing token quota limits helps control costs, encourage more efficient use of queries, and regulate system demands. In a multi-user configuration, token quota limits help provide equal access to all users ensuring everyone has an opportunity to submit queries.
```

Comment:
"system demands" -> "demand on the system"?

Reply:
fixed